examples/headers.rs

```diff
@@ -21,16 +21,11 @@ fn main() {
     // Match when we get `accept: */*` exactly.
     let accept_stars = warp::header::exact("accept", "*/*");
 
-    let index = host.and(accept_stars)
+    let routes = host.and(accept_stars)
```

Comment:
did this example lose its get check intentionally?

Reply:
Yea, the deprecation lint trigger, and as I started to adjust it, I figured showing method filters in the **headers** example was just noise.